src/gemm.jl

```diff
@@ -51,19 +51,8 @@ end
     Tropical(VectorizationBase._vbroadcast(a, content(s), si))
 end
 
-@inline function VectorizationBase.stridedpointer(A::AbstractArray{T}) where {T <: Tropical}
-    p, r = memory_reference(A)
-    stridedpointer(p, contiguous_axis(A), contiguous_batch_size(A), val_stride_rank(A), bytestrides(A), offsets(A))
-end
-
-@inline function VectorizationBase.stridedpointer(
-    ptr::Ptr{T}, ::StaticInt{C}, ::StaticInt{B}, ::Val{R}, strd::X, offsets::O
-) where {T<:Tropical,C,B,R,N,X<:Tuple{Vararg{Integer,N}},O<:Tuple{Vararg{Integer,N}}}
-    VectorizationBase.StridedPointer{T,N,C,B,R,X,O}(ptr, strd, offsets)
-end
-
 @inline function notropical(ptr::VectorizationBase.StridedPointer{Tropical{T},N,C,B,R,X,O}) where {T,N,C,B,R,X,O}
-    VectorizationBase.StridedPointer{T,N,C,B,R,X,O}(Ptr{T}(ptr.p), ptr.strd, ptr.offsets)
+    VectorizationBase.StridedPointer{T,N,C,B,R,X,O}(Ptr{T}(ptr.p), ptr.si)
```

Comment:
You could also call
```julia
stridedpointer(Ptr{T}(ptr.p), ptr.si, StaticInt{B}())
```
but this is fine.

Reply:
But I get:
```julia
mydot: Error During Test at /home/leo/.julia/dev/TropicalGEMM/test/gemm.jl:34
  Test threw exception
  Expression: mydot(a, b) ≈ transpose(a) * b
  MethodError: no method matching LayoutPointers.StridedPointer{Float64, 1, 1, 0, (1,), Tuple{StaticInt{8}}, Tuple{StaticInt{0}}}(::Ptr{Float64}, ::ArrayInterface.StrideIndex{1, (1,), 1, Tuple{StaticInt{8}}, Tuple{StaticInt{0}}}, ::StaticInt{0})
  Closest candidates are:
    LayoutPointers.StridedPointer{T, N, C, B, R, X, O}(::Any, ::Any) where {T, N, C, B, R, X, O} at /home/leo/.julia/packages/LayoutPointers/NXt9T/src/stridedpointers.jl:55
```